Automattic/themes
# Quadrat: fix featured images

#### Changes proposed in this Pull Request:

This PR makes featured images appear the default width of the content in both posts and pages by default.

This means that if a user wants a blank page template, it will need to be created.

#### To Test

- Add / edit a page and post
- Add a featured image
- Verify that it appears the default content width, along with the page title

#### Related issue(s):

Addresses #4063 and #4062
